<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸识别考勤系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 90, 100);
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 90, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>人脸识别考勤系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 90, 100);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ver 1.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/pic/logoface_3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>251</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_register">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>  人脸录入        </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon/pic/添加用户.png</normaloff>:/icon/pic/添加用户.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_checkin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>  人脸识别        </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/checkin.png</normaloff>:/icon/pic/checkin.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_checkin_record">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>  考勤记录        </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/checkinrecord.png</normaloff>:/icon/pic/checkinrecord.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_face_record">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>    数据库         </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/facerecord.png</normaloff>:/icon/pic/facerecord.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_users">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>      用户            </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/facerecord.png</normaloff>:/icon/pic/facerecord.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_about">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>      关于            </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/about.png</normaloff>:/icon/pic/about.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
    
	background-color: rgb(84, 110, 122);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}


QPushButton:hover
{
   
	background-color: rgb(127, 154, 167);

    border-radius:5px;
   
	
	color: rgb(255, 255, 255);
}

QPushButton:hover:pressed
{
    background-color:rgb(85,170,255);
    border:2px solid #3C80B1;
    border-radius:5px;
    color:white;
}</string>
       </property>
       <property name="text">
        <string>      退出            </string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icon/pic/exit.png</normaloff>:/icon/pic/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
